pontoon/actionlog/migrations/0004_auto_20240816_1045.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 4.2.11 on 2024-08-16 10:45
+
+import datetime
+
+from django.db import migrations
+
+
+def migrate_translation_to_actionlog(apps, schema_editor):
+    Translation = apps.get_model("base", "Translation")
+    ActionLog = apps.get_model("actionlog", "ActionLog")
+
+    translation_info = {
+        "date": (("translation:created", "Translation created"), "user"),
+        "approved_date": (
+            ("translation:approved", "Translation approved"),
+            "approved_user",
+        ),
+        "unapproved_date": (
+            ("translation:unapproved", "Translation unapproved"),
+            "unapproved_user",
+        ),
+        "rejected_date": (
+            ("translation:rejected", "Translation rejected"),
+            "rejected_user",
+        ),
+        "unrejected_date": (
+            ("translation:unrejected", "Translation unrejected"),
+            "unrejected_user",
+        ),
+    }
+
+    # date to end migration
+    end_date = datetime.datetime(
+        2020, 1, 7, 9, 25, 11, 829125, tzinfo=datetime.timezone.utc
+    )
+
+    actions_to_log = []
+
+    for translation in Translation.objects.filter(date__lt=end_date):
+        for attr, action_user in translation_info.items():
+            value = getattr(translation, attr)
```

Comment:
Now that we use `getattr()` twice, we should change the name of the variable to something more specific, e.g.:

```suggestion
            date = getattr(translation, attr)
```